### PR TITLE
fix(client): improve help modal verbiage and styles

### PR DESCRIPTION
**Pull request by raisedadead**
This PR adds friendlier verbiage in a non-direct tone. It also makes the warning part of the modal non-intimidating.

### CODE REVIEW

**Comment by raisedadead** [comment]
Screenshots for comparison:

<table>
<tr>
 <td>
image
 <td>
image
</table>

File: client/src/templates/Challenges/components/help-modal.tsx [hunk 1]
```diff
@@ -242,27 +246,30 @@ function HelpModal({
...
-            <p>
-              <Trans i18nKey='learn.tried-rsa'>
-                <a href={RSA} rel='noopener noreferrer' target='_blank'>
-                  placeholder
-                </a>
-              </Trans>
-            </p>
-            <div className='alert alert-danger'>
+            <div className='alert'>
               <FontAwesomeIcon icon={faExclamationCircle} />
```
Comment (naomi-lgbt) [inline client/src/templates/Challenges/components/help-modal.tsx:250]: Should we remove the icon too? I feel like without the warning callout this just feels out of place.

**Review by naomi-lgbt — APPROVED** [review]
Happy to land as-is, one question about the UI.

File: client/src/templates/Challenges/components/help-modal.tsx (hunk 1 above)
Comment (raisedadead) [reply client/src/templates/Challenges/components/help-modal.tsx:250]: Sure. I will add that to the change-set.

File: client/i18n/locales/english/translations.json [hunk 2]
```diff
@@ -1002,7 +1002,9 @@
     "user-agent": "User Agent is: <code>{{userAgent}}</code>",
     "challenge": "Challenge Information:",
     "whats-happening": "Tell us what's happening:",
-    "describe": "Describe your issue in detail here.",
+    "describe": "Describe your issue in detail here. You can ask us something like:",
+    "describe-details-line-1": "1. I am having trouble with the logic.",
+    "describe-details-line-2": "2. I tried to use a for loop but it is not working as expected.",
```
[the remark this answers is not in the document]
Comment (raisedadead) [reply client/i18n/locales/english/translations.json:1007]: We can check how it looks in real world, I will add this to the change-set.

**Comment by raisedadead** [comment]
image

**Comment by raisedadead** [comment]
I think while the text is too much, it is very good example of "how to ask meaningful" questions.

File: client/i18n/locales/english/translations.json [hunk 3]
```diff
@@ -1002,7 +1002,9 @@
     "user-agent": "User Agent is: <code>{{userAgent}}</code>",
     "challenge": "Challenge Information:",
     "whats-happening": "Tell us what's happening:",
-    "describe": "Describe your issue in detail here.",
+    "describe": "Describe your issue in detail here. You can ask us something like:",
+    "describe-details-line-1": "I'm getting this hint: \"Your h1 element should have an opening tag.\" But I think I already added it. Would you help me find what's missing in my code?",
```
[the remark this answers is not in the document]
Comment (raisedadead) [reply client/i18n/locales/english/translations.json:1006]: We could, but I am not sure how the behavior is when the text goes across the wire to the forum

**Comment by bbsmooth** [comment]
OK, a couple of accessibility related issues here (because of course you would all be disappointed if I didn't have something).

- Because the placeholder text is providing additional information, it must meet minimum WCAG contrast requirements of at least `4.5:1`. You can use the `::placeholder` pseudo-element to set the color of the text so that it will meet this requirement in all browsers. On the white background, `#767676` would get you `4.54:1`. On the dark background, `#7A7A7A` will get you `4.52:1`. This is covered under [WCAG 1.4.3 Contrast (Minimum)](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html).

- I would seriously consider reducing the amount of text in the placeholder. First, it's a lot of additional text that a screen reader user is going to have to listen to each time they want to create a post (at least until they figure out that it's the same text each time and so they stop the announcement early). Second, the text is being cut-off by the textarea and it's not always scrollable. For example, in FF on linux, the overflowing placeholder text does not cause a vertical scrollbar to appear. Note: The following pic is at 400% zoom to help emphasize this point, but it is also an issue at 100%.
fcc-no-scroll-bar
Notice that there is no visible scrollbar. Granted, it is possible to make the textarea taller so that you can read all of the placeholder text, but it is not possible to do this with the keyboard only (at least by default). And even with Chrome, which does display a vertical scrollbar when the placeholder text overflows, I don't seem to be able to scroll it with the keyboard only, and you also can't resize the textarea with the keyboard. In essence, keyboard only users are not going to be able to read all of the placeholder text unless you configure the textarea to always be tall enough to expose all of the placeholder text. I think these issues would fall under [WCAG 1.4.10 Reflow](https://www.w3.org/WAI/WCAG22/Understanding/reflow.html) and [WCAG 2.1.1 Keyboard](https://www.w3.org/WAI/WCAG22/Understanding/keyboard.html). The placeholder attribute is really intended to give a quick example of what input is expected. I understand what you are trying to do here, but in my opinion it might be asking a little too much of the placeholder attribute, unless you shorten it considerably.

**Comment by raisedadead** [comment]
Thanks @bbsmooth 

Your feedback is always appreciated. We will reduce the text length. 

> On the white background, #767676 would get you 4.54:1. On the dark background, #7A7A7A will get you 4.52:1.

Hey @huyenltnguyen 

Can you confirm the color codes above? That is do we have a existing variables I can use? Does it need changes in the underlying component? If yes, should it be a concern of a follow up PR?

**Comment by raisedadead** [comment]
image


Updated text as requested. I am only waiting on the text-color confirmation from Huyen - but I think that is a UI component level fix

**Review by huyenltnguyen — APPROVED** [review]
I'm going to merge this PR.

The remaining concern is tracked in https://github.com/freeCodeCamp/ui/issues/69.

**Pull request merged**